cloud-functions/functions/src/index.ts

```diff
@@ -440,3 +440,50 @@ export const initiatePasswordRecovery = functions.https.onCall((body) => {
       });
```

Comment:
Please apply `eslint` (`frontend/.eslintrc.js`) and `prettier` (`frontend/.prettierrc`) to this file, there are a bunch of formatting discrepancies, several let's that should be const's, etc.